### DISCUSSION TITLE
Resize Popup Player

### DISCUSSION THREAD

**Discussion by trymeouteh** [libre-tube/LibreTube#3424] [Ideas]
Please consider allowing the popup player to be resized. NewPipe has this with their Popup player which is very nice on large tablets.

**Comment by begin-theadventure** [comment]
The pop-up can be resized.

**Reply by trymeouteh** [reply]
How? I tried using my thumbs and it will not get bigger or smaller.

**Reply by Silther** [reply]
> The pop-up can be resized.

Do you mean the PiP player?

**Comment by Gabs-Unk** [comment]
I want it too, very much useful when you are doing something and need to watch something with pip player, like...if you are watching a tutorial some parts of text maybe display small so that you need to open main player in order to see
with a resizable player could help a lot
also good for someone while playing a game and have to wait haha, that way you increase the size could be good.
With that could also bring a option to minimize the pip player and keep a bubble to you open any time without have to open the app again.

**Comment by akano12** [comment]
You mean this? @trymeouteh 
 https://github.com/libre-tube/LibreTube/assets/50032603/9ec55e58-ba6e-489c-938a-57a73eb4356a

**Reply by trymeouteh** [reply]
Yeah, but I cannot resize it on my device with 2 fingers
